src/main/java/org/jenkinsci/maven/plugins/hpi/AbstractHpiMojo.java

```diff
@@ -791,37 +792,19 @@ public Reader getReader(Reader fileReader, Properties filterProperties) {
     private static void copyFilteredFile(File from, File to, String encoding, FilterWrapper[] wrappers,
                                          Properties filterProperties)
         throws IOException {
-        // buffer so it isn't reading a byte at a time!
-        Reader fileReader = null;
-        Writer fileWriter = null;
-        try {
-            // fix for MWAR-36, ensures that the parent dir are created first
-            Files.createDirectories(to.toPath().getParent());
-
-            if (encoding == null || encoding.length() < 1) {
-                fileReader = Files.newBufferedReader(from.toPath(), StandardCharsets.UTF_8);
-                fileWriter = Files.newBufferedWriter(to.toPath(), StandardCharsets.UTF_8);
-            } else {
-                FileInputStream instream = new FileInputStream(from);
-
-                FileOutputStream outstream = new FileOutputStream(to);
```

Comment:
There are some unused imports now that these have been deleted.

Reply:
addressed in ce934abf061132ca12cb609e54cc7954591ccaf0